### PR TITLE
[corona_stats] add countries charts

### PR DESCRIPTION
**Pull request by bhageena**
Screenshot from 2020-05-06 00-56-29

### CODE REVIEW

File: corona_stats/packages/corona_stats/corona_stats.js [hunk 1]
```diff
@@ -283,6 +286,16 @@ const success = (header, fields, footer, client) => {
...
     });
   }
+  const chart = {
+    type: 'image',
+    title: {
+      type: 'plain_text',
+      text:  country||'',
+      emoji: true
+    },
+    image_url: `https://raichand-8kehpaun1bf-apigcp.nimbella.io/countries/${encodeURI(country)}.png?${new Date().getTime()}`,
```
Comment (satyarohith) [inline corona_stats/packages/corona_stats/corona_stats.js:296]: What happens when `country` is empty?

Comment (rabbah) [reply corona_stats/packages/corona_stats/corona_stats.js:296]: Looks like it’s guarded on line 315.

**Review by rabbah — APPROVED** [review]
Lgtm

File: corona_stats/packages/corona_stats/corona_stats.js (hunk 1 above)
Comment (satyarohith) [reply corona_stats/packages/corona_stats/corona_stats.js:296]: Yes 😁

**Review by satyarohith — APPROVED** [review]
LGTM.

**Pull request merged**